oneanddone/tasks/filters.py

```diff
@@ -43,6 +43,11 @@ class TasksFilterSet(django_filters.FilterSet):
         label=_lazy(u'Search for tasks')
     )
 
+    difficulty = django_filters.MultipleChoiceFilter(
+        choices=((1, 'Beginner'), (2, 'Intermediate'), (3, 'Advanced')),
```

Comment:
You have hardcoded the values for `Beginner`, `Intermediate` and `Advanced` in this file, but they are also defined in the `Task` model. This is code duplication which will increase maintenance cost and could result in bugs. You should be able to get these values from the model, rather than defining them in this file. Please give that a try.